torsiondrive/td_api.py

```diff
@@ -274,32 +296,60 @@ def next_jobs_from_state(td_state, verbose=False):
 ### Utility functions for servers
 
 
-def create_initial_state(dihedrals, grid_spacing, elements, init_coords):
+def create_initial_state(dihedrals, grid_spacing, elements, init_coords, dihedral_ranges=None, energy_decrease_thresh=None, energy_upper_limit=None):
     """Create the initial input dictionary for torsiondrive API
 
     Parameters
     ----------
-    dihedrals : list of tuples
+    dihedrals : List of tuples
         A list of the dihedrals to scan over.
-    grid_spacing : list of int
+    grid_spacing : List of int
         The grid seperation for each dihedral angle
-    elements : list of strings
+    elements : List of strings
         Symbols for all elements in the molecule
-    init_coords : list of (N, 3) or (N*3) arrays
+    init_coords : List of (N, 3) or (N*3) arrays
```

Comment:
Using `list` is generally correct here unless you are using the Python 3.6 type annotations where you would have `List[np.ndarray]` to explicitly inform the type.

Reply:
@dgasmith Yeah I made the small change so it looks closer to the type hints but it's not strictly type hints yet.

Reply:
Right, my point here is that `List` is technically incorrect syntax as you explicitly reference the type `List` without an argument which would typically render as a syntax error if it wasn't a string.